datadreamer/dataset_annotation/owlv2_annotator.py

```diff
@@ -236,6 +236,22 @@ def annotate_batch(
                 torch.cat(all_labels), num_classes=len(prompts)
             )
 
+            # Fix the bounding boxes
+            width_ratio = 1
+            height_ratio = 1
+            width = images[i].width
+            height = images[i].height
+            if width > height:
+                height_ratio = height / width
+            elif height > width:
+                width_ratio = width / height
+
+            all_boxes = [
+                box
+                / torch.tensor([width_ratio, height_ratio, width_ratio, height_ratio])
```

Comment:
For me, multiplication works here instead of division. And maybe we should move this to a separate function in the class with a short description of the transformers bug?

Reply:
And does the division work for you as well or not? Because in the link with the solution the author is using division. But otherwise, yes, you're right.

Reply:
Only multiplication works for me